src/components/select-field/SelectDropdownList.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import type { SelectOptionValueProp } from './props';
+
+export const OVERLAY_SCROLLABLE_CLASS = 'bdl-SelectField-overlay--scrollable';
+
+type Props = {
+    children: React.Node,
+    isScrollable?: boolean,
+    multiple?: boolean,
+    placement: string,
+    popperRef: (?HTMLElement) => void,
+    scheduleUpdate?: () => void,
```

Comment:
Will there be a case when this is undefined?

Reply:
Shouldn't be, I'll remove the `?`